app/uk/gov/hmrc/hec/models/TaxDetails.scala

```diff
@@ -26,7 +26,8 @@ object TaxDetails {
   final case class IndividualTaxDetails(
     nino: NINO,
     sautr: Option[SAUTR],
-    taxSituation: TaxSituation
+    taxSituation: TaxSituation,
+    saIncomeDeclared: Option[IncomeDeclared]
```

Comment:
do none of the specs need updating as well or was this case class never actually being explicitly constructed in them?

Reply:
you are right, tbh, I didn't check, but there are references, I'll update

Reply:
fixed in commit - https://github.com/hmrc/hec/pull/13/commits/66558082d6d415065315c33cc6858aa304a94574